cluster-scope/base/podsecuritypolicy/metallb/metallb-system-controller.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: policy/v1beta1
+kind: PodSecurityPolicy
+metadata:
+  labels:
+    app: metallb
+  name: controller
+  namespace: metallb-system
```

Comment:
Should this resource have `metadata.namespace` ? I'm reading that this is a cluster-wide resource.

Reply:
Actually, that's a good question. Are PodSecurityPolicies cluster-wide?

Reply:
All the examples I can find suggest that yes, they are.